docs/snippets/modules/chains/popular/sqlite.mdx

```diff
@@ -40,15 +33,3 @@ const db = await SqlDatabase.fromDataSourceParams({
 If desired, you can return the used SQL command when calling the chain.
 
 <CodeBlock language="typescript">{SqlDBSqlOutputExample}</CodeBlock>
-
-## SAP Hana
-
-Here's an example of using the chain with a SAP HANA database:
-
-<CodeBlock language="typescript">{SqlDBSAPHANAExample}</CodeBlock>
-
-## Custom prompt
-
-You can also customize the prompt that is used. Here is an example prompting the model to understand that "foobar" is the same as the Employee table:
-
-<CodeBlock language="typescript">{SqlDBSqlCustomPromptExample}</CodeBlock>
```

Comment:
Deleted `SAP HANA` example as the implementation is the exact same (just swap out what DB to connect to) as above, and removed custom prompts since those are required for LCEL